### DISCUSSION TITLE
"HTTP Status 404 – Not Found" Unifi Script

### DISCUSSION THREAD

**Discussion by Hoofcott** [tteck/Proxmox#463] [Help Needed] [answered]
Hi there and thank you tteck for your scripts ! 

I encouter a problem with the Unifi. 
The script works well, the controller is OK and all.
But after a random piece of time, i have the "HTTP Status 404 – Not Found"error when trying to access its web page
The shell is still working but reboot, reinstall, nothing unlock the error...

If someone has an idea about this?
Maybe a log file may help but i don't know which one?

**Comment by tteck** [comment, marked as answer]
https://github.com/tteck/Proxmox/issues/451

**Reply by Hoofcott** [reply]
Oh sorry, i promise i've searched but not with the right keywords...
Seems working for now, i'll update if needed, thank you !
